rediscluster/client.py

```diff
@@ -285,7 +285,10 @@ def _determine_slot(self, *args):
                 raise RedisClusterException("{0} - all keys must map to the same key slot".format(command))
             return slots.pop()
 
-        key = args[1]
+        if command == 'OBJECT':
```

Comment:
A very small thing, but i would prefer it to be this

```
key = args[1]

if command = 'OBJECT':
    key = args[2]
```

Alter this and i merge this PR after that.